### PR TITLE
Added spending logs &  transparency report of spending for 2022-Q1 and 2022-Q2

### PR DESCRIPTION
**Pull request by cekickafa**
There is a missing Btc description for transaction with spend_id:8 at 2022-02-27 1738$.

### CODE REVIEW

**Review by MCM-Mike — APPROVED** [review]
I did check a few transaction by hand and they match. 
Therefore I can approve it.

**Comment by cekickafa** [comment]
> I did check a few transaction by hand and they match. Therefore I can approve it.
Thanks for review.

File: pending-payments.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+# Pending Payments
+
+|Requester|Payment for|Amount|Date|Currency |SMA |Approved|Status
+| ----- | ----- | ----- | ----- | ----- | ----- | ----- | ----- |
+|@dtavarez|Mining farm vpn cost|13.89 $|16.08.2022|**204.525 GRIN**|0.06791 $|Yes|**Pending** |
+|@dtavarez|Aug-December funding request|3.750 €|11.22.2022|**86.855 GRIN**|0.4317|Yes|**Pending** |
```
Comment (future3OOO) [inline pending-payments.md:7]: | 3750 €| 86,855.115 GRIN | Paid 11/24

**Review by future3OOO — CHANGES_REQUESTED** [review]

File: pending-payments.md (hunk 1 above)
Comment (cekickafa) [reply pending-payments.md]: @future3OOO edited as '**Paid**'

**Review by Anynomouss — APPROVED** [review]
I found no problems with the report, but I would propose to simplify by making it yearly, not quarterly. Instead of reporting changes in balance per quarter, we can provide a graph of changes in value hold in BTC and USD using this handy website. I removes all the manual labor of calculating changes in Grin and BTC for a certain period:
https://bitinfocharts.com/bitcoin/address/bc1qmdhmgmhd6j89225hzdh7dxqgmen3y2q0g4vgpez0tw9tkp4ae39qsqvuyl

For the reporting I recommend changing from reporting per quarter to just yearly. Show all incoming and outgoing, or begin value at the year 2022 and end value at the end of 2022, since there are many fluctuation with incoming miner payments etc. Only mention donations that came in separately as total amount of  received donations in 2022 for Grin.  In other words, keep it as simple as possible, just a table with a small summary and separate mention of donations.

**Comment by cekickafa** [comment]
I will summarize it for 2022. 

> I found no problems with the report, but I would propose to simplify by making it yearly, not quarterly. Instead of reporting changes in balance per quarter, we can provide a graph of changes in value hold in BTC and USD using this handy website. I removes all the manual labor of calculating changes in Grin and BTC for a certain period: https://bitinfocharts.com/bitcoin/address/bc1qmdhmgmhd6j89225hzdh7dxqgmen3y2q0g4vgpez0tw9tkp4ae39qsqvuyl
> 
> For the reporting I recommend changing from reporting per quarter to just yearly. Show all incoming and outgoing, or begin value at the year 2022 and end value at the end of 2022, since there are many fluctuation with incoming miner payments etc. Only mention donations that came in separately as total amount of received donations in 2022 for Grin. In other words, keep it as simple as possible, just a table with a small summary and separate mention of donations.

I will summarize as 2022 and simplfy it.

**Review by Anynomouss — APPROVED** [review]

**Review by mojitoo — APPROVED** [review]

**Review by davidtavarez — APPROVED** [review]

**Pull request merged**